cmd/ocm-backplane/managedJob/createManagedJob.go

```diff
@@ -335,3 +389,53 @@ func getJobStatus(client BackplaneApi.ClientInterface, job *BackplaneApi.Job) (B
 
 	return *formatJobResp.JSON200.JobStatus.Status, nil
 }
+
+var listDynaKube = func(cl client.Client,ctx context.Context, u client.ObjectList, opts ...client.ListOption) error{
+	return cl.List(ctx, u, opts...)
+}
+
+func getDynatraceInstance() (instance string, err error) {
+	bp, err := config.GetBackplaneConfiguration()
+	if err != nil {
+		return
+	}
+	cfg, err := login.GetRestConfig(bp, options.clusterID)
+	if err != nil {
+		return
+	}
+	cl, err := client.New(cfg, client.Options{})
+
+	if err != nil {
+		return
+	}
+	u := &unstructured.Unstructured{}
+	u.SetGroupVersionKind(schema.GroupVersionKind{
```

Comment:
just to confirm, we do not need additional permission to list this?

Reply:
@bmeng We tested it with @AlexVulaj 's id and we did not need any additional permissions. @AlexVulaj  please confirm.